lazy.config.ts

```diff
@@ -79,7 +79,10 @@ const config = {
 			runsAfter: { 'build-types': {} },
 			cache: {
 				inputs: {
-					exclude: ['*.tsbuildinfo'],
+					exclude: [
+						'*.tsbuildinfo',
+						`<rootDir>/templates/*/package.json`,
```

Comment:
@ds300 this isn't skipping the template folders like i'd like to, what am i doing wrong here?

Reply:
eh nvm, gonna do this a different way